final_assessment/task_4.py

```diff
@@ -0,0 +1,7 @@
+None of the functions are returned at the end of each function.
```

Comment:
3/25 - Good points made, however too few to get 25 marks. You received marks here for the mention of refactoring and given examples.